### PR TITLE
Update dependencies and lint

### LINKED ISSUE #3182: `format-discussion-titles`: Cannot read property 'parentElement' of null
image

[github.com-1591371193237.log](https://github.com/sindresorhus/refined-github/files/4737002/github.com-1591371193237.log)

### PR DESCRIPTION
**Pull request by yakov116**
- Remove optional chaining from `show-user-top-repositories` 
	Not needed its only ran on one page and it should error if it does not exist
- Hide useless void from `hide-useless-comments`
- Add return false to `cross-deleted-pr-branches`
- Switch `enable-file-links-in-compare-view` and `revert-file` to toggle
- Change `fast-reviews` wording to review now - As per #3149 (comment)
- Update `github-url-detection` - Fixes #3182
- Update all deprecated `github-url-detection` detections

### CODE REVIEW

File: source/features/raw-file-link.tsx [hunk 1]
```diff
@@ -4,28 +4,26 @@ import delegate from 'delegate-it';
...
-		<a href={url.join('/')} className="pl-5 dropdown-item btn-link rgh-raw-file-link" role="menuitem">
+		<a href={href} className="pl-5 dropdown-item btn-link rgh-raw-file-link" role="menuitem">
 			View raw
 		</a>
 	);
 }
 
 function init(): void {
-	delegate(document, '.js-file-header-dropdown > summary', 'click', handleMenuOpening);
+	delegate(document, '.js-file-header-dropdown > summary:not(.rgh-raw-file-link)', 'click', handleMenuOpening);
```
Comment (fregante) [inline source/features/raw-file-link.tsx:26]: I’m not sure why this doesn’t listen to `toggle` instead

File: source/features/raw-file-link.tsx [hunk 2]
```diff
@@ -4,28 +4,26 @@ import delegate from 'delegate-it';
...
-	if (select.exists('.rgh-raw-file-link', dropdown)) {
-		return;
-	}
-
 	const viewFile = select<HTMLAnchorElement>('[data-ga-click^="View file"]', dropdown)!;
-	const url = viewFile.pathname.split('/');
-	url[3] = 'raw'; // Replaces 'blob'
+	const href = new GitHubURL(viewFile.href).assign(
+		{route: 'raw'}
+	).toString();
```
Comment (fregante) [inline source/features/raw-file-link.tsx:16]: Can be one line. Also you can drop `toString` and just use destructuring:

const {href} = etc

File: source/features/show-user-top-repositories.tsx [hunk 3]
```diff
@@ -16,7 +16,7 @@ function buildUrl(queryField: string): URL {
 
 function init(): void {
 	// Showcase title
-	select('.js-pinned-items-reorder-container .text-normal')?.firstChild!.after(
+	select('.js-pinned-items-reorder-container .text-normal')!.firstChild!.after(
```
Comment (fregante) [inline source/features/show-user-top-repositories.tsx:19]: I think this was necessary. Profiles might not have pinned items

Comment (yakov116) [reply source/features/show-user-top-repositories.tsx:19]: Seems not https://github.com/goastat

File: source/features/raw-file-link.tsx (hunk 1 above)
Comment (yakov116) [reply source/features/raw-file-link.tsx:26]: Just tried it, seems not to work

Comment (fregante) [reply source/features/raw-file-link.tsx:24]: Summary doesn't `toggle`, you'll have to update the selector. Look at other features

Comment (yakov116) [reply source/features/raw-file-link.tsx:24]: I can look but `enable-file-links-in-compare-view` uses the same thing

Comment (yakov116) [reply source/features/raw-file-link.tsx]: Done should I do this to `enable-file-links-in-compare-view`

File: source/features/raw-file-link.tsx [hunk 4]
```diff
@@ -7,10 +7,10 @@ import features from '.';
 import GitHubURL from '../github-helpers/github-url';
 
 function handleMenuOpening(event: delegate.Event): void {
```
Comment (fregante) [inline source/features/raw-file-link.tsx:9]:
```suggestion
function handleMenuOpening({delegateTarget: dropdown}: delegate.Event): void {
```

File: source/features/raw-file-link.tsx (hunk 1 above)
Comment (yakov116) [reply source/features/raw-file-link.tsx]: Did it

**Comment by fregante** [comment]
You can re-revert e8968aa separately. This became a cleanup PR

**Comment by yakov116** [comment]
Sure

**Comment by fregante** [comment]
I'll leave this open if you want to cleanup further. Nu rush

**Comment by yakov116** [comment]
It will cause a conflict and I (think) am done. (Unless you had something in mind you wanted done here)

File: source/features/latest-tag-button.tsx [hunk 5]
```diff
@@ -109,8 +108,8 @@ async function init(): Promise<false | void> {
 		link.append(' ', <span className="css-truncate-target">{latestTag}</span>);
 	}
 
-	link.classList.add('tooltipped', 'tooltipped-ne');
```
Comment (yakov116) [inline source/features/latest-tag-button.tsx:112]: I moved this into the if statement's since it can be blank

Comment (fregante) [reply source/features/latest-tag-button.tsx]: This would be a bugfix, not linting anymore. New PR is ok

**Comment by fregante** [comment]
# 🙌 

Looking good!

**Pull request merged**